### PR TITLE
[new release] dune (1.3.0)

### PR DESCRIPTION
**Pull request by rgrinberg**
CHANGES:

- Support colors on Windows (ocaml/dune#1290, @diml)

- Allow `dune.configurator` and `base` to be used together (ocaml/dune#1291, fix
  ocaml/dune#1167, @diml)

- Support interrupting and restarting builds on file changes (ocaml/dune#1246,
  @kodek16)

- Fix findlib-dynload support with byte mode only (ocaml/dune#1295, @bobot)

- Make `dune rules -m` output a valid makefile (ocaml/dune#1293, @diml)

- Expand variables in `(targets ..)` field (ocaml/dune#1301, ocaml/dune#1320, fix ocaml/dune#1189, @nojb,
  @rgrinberg, @diml)

- Fix a race condition on Windows that was introduced in 1.2.0
  (ocaml/dune#1304, fix ocaml/dune#1303, @diml)

- Fix the generation of .merlin files to account for private modules
  (@rgrinberg, fix ocaml/dune#1314)

- Exclude the local opam switch directory (`_opam`) from the list of watched
  directories (ocaml/dune#1315, @dysinger)

- Fix compilation of the module generated for `findlib.dynload`
  (ocaml/dune#1317, fix ocaml/dune#1310, @diml)

### CODE REVIEW

**Comment by mseri** [comment]
The PR is in opam 1.2 format. Can you update it to opam2?

File: packages/dune/dune.1.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
```
Comment (avsm) [inline packages/dune/dune.1.3.0/opam]: This needs to be a `2.0` format PR for the master branch

**Comment by emillon** [comment]
@rgrinberg the steps to convert are:
- run `opam-package-upgrade ./opam`
- convert the `url` file as described in #12653
- amend, force push

**Comment by rgrinberg** [comment]
@emillon I was planning to just re-submit with the new dune-release. It should work with opam 2.0

**Comment by rgrinberg** [comment]
Okay, I think it's using the 2.0 format now.

**Pull request merged**